spec/ruby/language/predefined_spec.rb

```diff
@@ -559,10 +559,20 @@ def foo
     $VERBOSE = @verbose
   end
 
-  it "can be assigned a String" do
-    str = "abc"
-    $/ = str
-    $/.should equal(str)
+  ruby_version_is ""..."3.5" do
+    it "can be assigned a String" do
+      str = "abc"
+      $/ = str
+      $/.should equal(str)
+    end
+  end
+
+  ruby_version_is "3.5" do
+    it "freezes the assigned String" do
+      str = +"abc"
+      $/ = str
+      $/.frozen?.should == true
```

Comment:
I think `Expected "abc".frozen?` is better than `Expected false == true`.
And whether the string is assigned should be tested.

```suggestion
      $/.should.frozen?
      $/.should == str
```

Reply:
Thank you!